exec-unittests

```diff
@@ -132,31 +126,26 @@ _subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-sc
 let count=0
 for _major in ${major_list}; do
     if [[ "${_major}" == "agent" ]]; then
-        # The parallel program is really cool, but what a pain that it posts this
-        # citation request.
-        if [[ -z "${HOME}" ]]; then
-            # If `HOME` is not set, the `parallel` program looks here. Don't ask.
-            _para_dir=/tmp/.parallel
-        else
-            _para_dir=${HOME}/.parallel
-        fi
-        mkdir -p ${_para_dir}
-        touch ${_para_dir}/will-cite
+        # The parallel program is really cool.  The usage of `parallel` is
+        # internal and automated; we are not disturbing product code to depend
+        # on this tool, and we, as developers, have viewed the citation and are
+        # justified in suppressing future displays of it in our development
+        # processes (use of --will-cite below).
```

Comment:
s/disturbing/distributing/  😁

Also, I would change "to depend" to "which depends".